leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -445,6 +449,17 @@ public void showLocations(List<Address> addresses) {
   }
 
   @Override
+  public void showDebouncedLocations(List<Address> addresses) {
+    if (addresses != null) {
+      fillLocationList(addresses);
+      if (!addresses.isEmpty()) {
```

Comment:
Maybe a method to filter addresses could be great (if anyone overrides activity)